docs/PULL_REQUEST_TEMPLATE

```diff
@@ -0,0 +1,9 @@
+### Description
+
+[LP-](https://philanthropyu.atlassian.net/browse/LP-)
```

Comment:
What if PR handles more than two stories

Reply:
It just fills the box with this template when you create PR, So if PR contains 2 stories you can copy paste this link..